pkg/forwarder/endpoints/endpoints.go

```diff
@@ -48,6 +48,8 @@ var (
 	ConnectionsEndpoint = transaction.Endpoint{Route: "/api/v1/collector", Name: "connections"}
 	// OrchestratorEndpoint is a v1 endpoint used to send orchestrator checks
 	OrchestratorEndpoint = transaction.Endpoint{Route: "/api/v1/orchestrator", Name: "orchestrator"}
+	// OrchestratorManifestEndpoint is a v1 endpoint used to send orchestrator manifests
+	OrchestratorManifestEndpoint = transaction.Endpoint{Route: "/api/v1/orchmanifest", Name: "orchmanifest"}
```

Comment:
Any preference for the naming here 🤔 ?

Reply:
please take the max intake length into consideration + the related rfc for intake for EP backend (api/v2)